scripts/devops_tasks/common_tasks.py

```diff
@@ -20,14 +20,16 @@
 import io
 import re
 import pdb
-
-# ssumes the presence of setuptools
-from pkg_resources import parse_version, parse_requirements
+from enum import Enum
+# Assumes the presence of setuptools
+from pkg_resources import parse_version, parse_requirements, Requirement
 
 # this assumes the presence of "packaging"
 from packaging.specifiers import SpecifierSet
 from packaging.version import Version
 
+DEV_REQ_FILE = "dev_requirements.txt"
+NEW_DEV_REQ_FILE = "new_dev_requirements.txt"
 
 logging.getLogger().setLevel(logging.INFO)
 
```

Comment:
Do we use `MANAGEMENT_PACKAGE_IDENTIFIERS` anywhere anymore? If you don't see a single python script referencing it, we should blow it away.

Reply:
It's used to handle error code 5 gracefully for tox tests